elastic/kibana
# [ResponseOps][Alerts] Fix DSL filter issues in search bar

## Summary

Resolves https://github.com/elastic/kibana/issues/183908
Resolves https://github.com/elastic/kibana/issues/192557

This PR fixes an issue of DSL filter in
<details><summary>Search bar in Alerts page of Stack management</summary>
<img width="1237" alt="image" src="https://github.com/user-attachments/assets/b9b380d2-80a7-4754-95f2-6b6831923c4d">
</details> 

<details><summary>Search bar in Alerts page of Observability</summary>
<img width="1237" alt="image" src="https://github.com/user-attachments/assets/68ceb97f-b958-47f4-b356-757cbafd9170">
</details> 

<details><summary>Search bar in "If alerts matches query" action filter</summary>
<img width="1281" alt="image" src="https://github.com/user-attachments/assets/1f99ca43-c4b5-4f52-b50f-914f1e205c5b">
</details> 

<details><summary>Search bar in Maintenance window page</summary>
<img width="1272" alt="image" src="https://github.com/user-attachments/assets/ffaa317d-c14b-45a2-9d02-00f7a10239ab">
</details> 


### Checklist

### How to verify
- Create a DSL filter in above mentioned pages
- Verify that filter is applied properly
